### DISCUSSION TITLE
How to view multiple epub frontpages in Explorer

### DISCUSSION THREAD

**Discussion by JanR-lang** [sumatrapdfreader/sumatrapdf#3523] [Documentation]
I see previews of frontpages of pdf-files in Windows Explorer, but no possibilities for epub.
In stead of Explorer it may also be in similar app, like [
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/135434137/5bcdf6b4-bab3-4423-a13a-5fdc4b7eecc8)
](url).
Besides (though less important) I also do not see frontpages of Word, Excel. Powerpoint (only the generic logo with W, X, P).

How do I include the necessary settings?

**Comment by GitHubRulesOK** [comment]
For MS files like word excel powerpoint you need a Microsoft Viewer such as comes with MS Office.
For ePub there are several including one copied from SumatraPDF but perhaps a good one from GitHub may be Dark Thumbs ? It only works in the main folder thumbnail area not as far as I know the Pre-Viewer Pane see https://github.com/fire-eggs/DarkThumbs
